static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -473,6 +518,8 @@ class GroupDetails extends Component<Props, State> {
       GroupStore.loadInitialData([data]);
 
       if (trackView) {
+        // make sure releases have loaded before we track the view
+        await groupReleasePromise;
         this.trackView(project);
```

Comment:
```suggestion
        groupReleasePromise.then(() =>  this.trackView(project));
```

i don't think we want this to go into the handleRequestError if the request response fails. Not sure what would happen